data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -188,6 +188,14 @@ local raid(level='raid0', uefi=false) = {
   lvm_tpm_fde: lvm(true, 'tpmFde'),
   raid0: raid('raid0'),
   raid0_uefi: raid('raid0', true),
+  raid1: raid('raid1'),
+  raid1_uefi: raid('raid1', true),
+  raid5: raid('raid5'),
+  raid5_uefi: raid('raid5', true),
+  raid6: raid('raid6'),
+  raid6_uefi: raid('raid6', true),
+  raid10: raid('raid10'),
+  raid10_uefi: raid('raid10', true),
```

Comment:
we don't need uefi for all of them, just raid0 (and that is not for raid testing, but for the other feature to edit existing raid), in general having some minimal test coverage of uefi is enough.